lib/fluent/plugin/out_mongo.rb

```diff
@@ -157,6 +158,11 @@ def configure(conf)
       configure_logger(@mongo_log_level)
 
       log.debug "Setup mongo configuration: mode = #{@tag_mapped ? 'tag mapped' : 'normal'}"
+
+      (@date_keys || []).each { |field_name|
```

Comment:
Following code is better. Don't put log for `nil` date_keys case

```
if @date_keys
  @date_keys.each { |field_name|
    @accessors[field_name] = record_accessor_create(field_name)
  }
  log.debug "Setup record accessor for every date key"
end
```

Reply:
You are right.. Implemented in latest commit